tests/data/github/repos.py

```diff
@@ -60,69 +61,6 @@
             'url': 'https://github.com/example_org',
             'login': 'example_org', '__typename': 'Organization',
         },
+        'requirements': {'text': 'cartography==0.1.0\nhttplib2>=0.7.0\njinja2\nlxml\n'},
     },
 ]
-
-TRANSFORMED_REPOS_DATA = {
```

Comment:
I removed this because it's unfeasible to maintain and doesn't add much value in testing; we want to test the whole set of ETL steps anyway.